context/app/static/js/pages/Cells/Cells.jsx

```diff
@@ -1,66 +1,53 @@
 import React, { useState } from 'react';
 
-import SectionHeader from 'js/shared-styles/sections/SectionHeader';
-import { Alert } from 'js/shared-styles/alerts';
 import AccordionSteps from 'js/shared-styles/accordions/AccordionSteps';
 import DatasetsSelectedByExpression from 'js/components/cells/DatasetsSelectedByExpression';
 import DatasetsTable from 'js/components/cells/DatasetsTable';
-
-function About(props) {
-  const { title, given, returns } = props;
-  return (
-    <>
-      <SectionHeader>{title}</SectionHeader>
-      <p>
-        <b>Given</b> {given}
-      </p>
-      <p>
-        <b>Returns</b> {returns}
-      </p>
-    </>
-  );
-}
+import QuerySelect from 'js/components/cells/QuerySelect';
 
 function Cells() {
   const [results, setResults] = useState([]);
   const [minExpressionLog, setMinExpressionLog] = useState(1);
   const [minCellPercentage, setMinCellPercentage] = useState(10);
-  const [geneNames, setGeneNames] = useState([]);
+  const [cellVariableNames, setCellVariableNames] = useState([]);
+  const [queryType, setQueryType] = useState([]);
 
   return (
-    <>
-      <Alert severity="warning">UI is still being designed, but we want to demonstrate that the API works.</Alert>
-      <About
-        title="Datasets selected by gene expression"
-        given="a list of genes, a minimum expression level, and a minimum percentage of cells at that expression level"
-        returns="a list of UUIDs for datasets which meet those minimums"
-      />
-      <AccordionSteps
-        steps={[
-          {
-            heading: 'Parameters',
-            content: (
-              <DatasetsSelectedByExpression
-                setResults={setResults}
-                minExpressionLog={minExpressionLog}
-                setMinExpressionLog={setMinExpressionLog}
-                minCellPercentage={minCellPercentage}
-                setMinCellPercentage={setMinCellPercentage}
-                geneNames={geneNames}
-                setGeneNames={setGeneNames}
+    <AccordionSteps
+      steps={[
+        {
+          heading: '1. Query Type',
+          content: <QuerySelect setQueryType={setQueryType} setCellVariableNames={setCellVariableNames} />,
```

Comment:
Made a similar comment elsewhere, but maybe a simpler datastructure for this. I guess I start to get nervous with nested context changes in JSX, but not worth a big refactor.